Conference website

ginabeki opened the pull request:
### Conference Website for Art Week Event
I performed the following things in this task;

- Built the home page.
- Built the about page and linked it to the home page.
- Created the two pages in two different versions screen sizes:
  - Mobile: up to 768px wide.
   - Desktop: 768px or wider.
- I also added Interactions on the page by linking the menu to the about page and header to the home page
- On mobile, I ensured that when the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- When a user clicks the (X) button, the menu closes.
- Created featured speakers dynamically using JS.
Here's a [link](https://www.loom.com/share/1186e713f5884bf2bcc7b744adeba7f7) to my presentation

ginabeki committed:
Set up sass

ginabeki committed:
Set up css as an output after sass compilation

ginabeki committed:
Assets for the project

ginabeki committed:
Markup for the entire page

ginabeki committed:
Set up markup for the about section

ginabeki committed:
Linters configuration

ginabeki committed:
Update home page

ginabeki committed:
Update desktop

ginabeki committed:
Add about section

ginabeki committed:
Add about section

ginabeki committed:
Update images for the page

ginabeki committed:
Update sass module files

ginabeki committed:
Add speaker's section dynamically

ginabeki committed:
Output file for sass

ginabeki committed:
Add section for speaker's team

ginabeki committed:
Fix linters

ginabeki committed:
Dependency files for sass environment

ginabeki committed:
Update documentation

ginabeki committed:
Screenshot image for the live website

ginabeki committed:
Fix linters

ginabeki committed:
Fix contrast issues to improve accessibility on the page

ginabeki committed:
Fix linters error

ginabeki committed:
Add z-index on nav bar

ginabeki committed:
Fix linters

ginabeki committed:
Update README.md

zieeco commented on README.md:
```diff
@@ -1 +1,85 @@
+### Live Demo
+
+- To view a live demo of the project you can click on this [link](https://ginabeki.github.io/Conference-Website/).
+
```
- Hey, please let's add the link to the video presentation here in the readme. Also, your video is longer than 5 min in length as this is meant to not exceed 5 min long according to the project requirements. Please, kindly record another one with a maximum length of not more than 5 mins long to meet the project requirement.

zieeco commented on about.html:
```diff
@@ -0,0 +1,83 @@
+    </header>
+    <section class="hero-about">
+      <h3 class="about-text">"Hello, Creative World!"</h3>
+      <h1 class="about-detail">Explore Kenyan Art</h1>
```
- Hey, To enforce the best practices in using the html5 semantic tag, a webpage is only meant to have one h1 which denotes the main heading in the webpage. You can use another header tag here as your main heading is in the index.html file. Please update this to improve the web accessibility of your page.

zieeco commented on css/styles.css:
```diff
@@ -0,0 +1,602 @@
+  gap: 15px;
+}
+
+.team-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```
- [ ] Hey, please let's improve our design in the `mobile version` a little to be close to the design template as the aim of this project is to help make a total clone of an existing design, thereby paying attention to details is a skill that will make you stand out in the job market. see the suggestion below:

*original design*
![Screenshot from 2022-10-19 08-43-13](https://user-images.githubusercontent.com/59397159/196631660-d3e4e2a8-ea62-48fb-ae72-2c425966cee2.png)

*your design*
![Screenshot from 2022-10-19 08-45-03](https://user-images.githubusercontent.com/59397159/196631649-5261937a-162d-48fc-83a3-3d85823dc183.png)

*your design after I edited*
![Screenshot from 2022-10-19 08-52-53](https://user-images.githubusercontent.com/59397159/196631645-ad0b13f7-a48f-47d0-8a33-a83419b994b8.png)

- As you can from those designs, the content on your design is not that close compared to the original design
   - You can reduce the font size,
   - Remove the gap in `.team-details`,
   - You can make the height/width of the image `8rem`,
   - You can also reduce the description of the team members,*[If you do this, your design will be very close :100:`]*
   - All these are just little tweaks around to improve the design.
- [x] Please, make a few adjustments to improve your design and also, to improve the user experience of your website.

zieeco commented on css/styles.css:
```diff
@@ -0,0 +1,602 @@
+  align-items: center;
+}
+
+.card {
+  background-color: #b3b3b351;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0.5rem;
+}
```
- Hey, I really like your design. well done! I just want to point out something. Please, let's try to be attentive to details. This skill is very important in the dev world. Your card is 3, while the design is 5, can we add more cards to match the number? Please, I encourage you to do this.

![Screenshot from 2022-10-19 09-16-00](https://user-images.githubusercontent.com/59397159/196636292-26a33a63-4cd3-45d5-83fb-3bfdc50aeae1.png)

zieeco replied:
- Also, the hover color of the card is `white`, let's update this also.

ginabeki replied:
Done

zieeco commented on index.html:
```diff
@@ -0,0 +1,117 @@
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+
+        <div class="logo">
+          <a href="./index.html">
+            <img src="./images/logo.png" alt="Company logo" />
+          </a>
+        </div>
+        <ul class="menu">
+          <li class="nav-link"><a href="./about.html">About</a></li>
+          <li class="nav-link"><a href="">Program</a></li>
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
```
- Hey, you're doing great! Please, your header is missing the first top nav. Please, let's add this match the design as much as we can below:

*original design*
![Screenshot from 2022-10-19 09-22-21](https://user-images.githubusercontent.com/59397159/196637819-06992e2e-ef21-4ed8-9aa1-f0c4be16cd07.png)

*your design*
![Screenshot from 2022-10-19 09-21-36](https://user-images.githubusercontent.com/59397159/196637827-73d92cec-97ba-49b2-86f9-dafa180acc04.png)

ginabeki replied:
Done

zieeco commented on about.html:
```diff
@@ -0,0 +1,83 @@
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+
+        <div class="logo">
+          <a href="./index.html">
+            <img src="./images/logo.png" alt="Company logo" />
+          </a>
+        </div>
+        <ul class="menu">
+          <li class="nav-link"><a href="/index.html">Home</a></li>
+          <li class="nav-link"><a href="">Program</a></li>
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
```
- Hey, you're doing great! Please, your header is missing the first top nav. Please, let's add this match the design as much as we can below:

*original design*
![Screenshot from 2022-10-19 09-22-21](https://user-images.githubusercontent.com/59397159/196637819-06992e2e-ef21-4ed8-9aa1-f0c4be16cd07.png)

*your design*
![Screenshot from 2022-10-19 09-21-36](https://user-images.githubusercontent.com/59397159/196637827-73d92cec-97ba-49b2-86f9-dafa180acc04.png)

zieeco commented on css/styles.css:
```diff
@@ -0,0 +1,602 @@
+    text-align: left;
+  }
+
+  .workshop {
+    flex-direction: row;
+  }
```
- Hey, to control the width of each workshop, let's add a widht of about 65% to almost be close to the design template, thereby aligning the card in the center below:

![Screenshot from 2022-10-19 09-32-31](https://user-images.githubusercontent.com/59397159/196640518-6435fcb5-62e7-435f-9a6b-8b9fd448fab8.png)

zieeco commented on css/styles.css:
```diff
@@ -0,0 +1,602 @@
+.workshop-2 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-2.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
```
- Hey, please, let's increase the height to 12rem so our design is close to match the design template.

zieeco commented on css/styles.css:
```diff
@@ -0,0 +1,602 @@
+.workshop-1 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-1.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
```
- Hey, please, let's increase the height to 12rem so our design is close to matching the design template.

zieeco reviewed (changes requested):
Hi @ginabeki,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good code organisation :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ginabeki committed:
Update layout page

ginabeki committed:
Update the variable for image src

ginabeki committed:
Update links for the menu items

ginabeki committed:
Output file for the styles

ginabeki committed:
Update markup for the nav links

ginabeki committed:
Add assets to the page

ginabeki committed:
Update README.md

ginabeki commented:
The following changes were made;
- Added a new loom [live link](https://www.loom.com/share/87858677e9c740abbdf4e7646deae94a) that's within the required length.
- Removed extra h1 tag on the page to improve SEO
- Fixed the speaker's section to match the design
- Changed hover to white on the cards
- Added top nav for the social media links
- Fixed the width and height of the workshop images

medaminedev66 commented on about.html:
```diff
@@ -0,0 +1,95 @@
+        Wamae.
+      </p>
+      <p class="contact">Please contact us for any further questions at <a href="https://github.com/ginabeki">artsy@studio.com</a> </p>
+    </section>
```
[OPTIONAL] Your layouts on the about page cross each other. I suggest addressing that so your UI design is better.

<img width="200" alt="image" src="https://user-images.githubusercontent.com/77190203/196686546-776b2363-4017-4756-92f2-1c74e01c02f0.png">

medaminedev66 commented on README.md:
```diff
@@ -1 +1,86 @@
+### Live Demo
+- Here's the [live link](https://www.loom.com/share/87858677e9c740abbdf4e7646deae94a) representation on loom
+
+- To view a live demo of the project you can click on this [link](https://ginabeki.github.io/Conference-Website/).
```
[OPTIONAL] The project is deployed properly. Great job 👏. But in the live demo, I'm still not able to navigate to the home page from the about page. I recommend fixing this issue to improve the user experience.

ginabeki replied:
@medaminedev66 Thank you for the review, I'll go through the optional suggestions and fix the remaining few areas

medaminedev66 reviewed (approved):
Hi @ginabeki 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The video presentation contains a clear and objective description of the project.
- Your design looks great and matches the provided design.

### Optional suggestions
- Check the comments below the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
